### PR TITLE
change header for platform per TOC guidance

### PR DESCRIPTION
**Pull request by jacob-delgado**
Change header for untested docs

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/setup/platform-setup/azure/index.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ test: no
 
 This page was last updated September 12, 2019.
 
-{{< boilerplate untested-platform >}}
+{{< boilerplate untested-documentt >}}
```
Comment (ericvn) [inline content/en/docs/setup/platform-setup/azure/index.md:16]: Let fix is to remove the extra t
```suggestion
{{< boilerplate untested-document >}}
```

File: content/en/boilerplates/untested-document.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+---
+{{< warning >}}
+This document has not been tested on the Istio 1.8 release and may contain bugs.
```
Comment (ericvn) [inline content/en/boilerplates/untested-document.md:4]: Should this be a 1.9 statement instead of 1.8 (although we won't have tested on either)?

Comment (jacob-delgado) [reply content/en/boilerplates/untested-document.md]: Changed. I will do a manual cherry pick into 1.8 when this gets merged.

**Review by ericvn — APPROVED** [review]

**Pull request merged**